plugins/apple/swift/v1.20.2/Dockerfile

```diff
@@ -1,12 +1,12 @@
 # syntax=docker/dockerfile:1.4
-FROM swift:5.7.0-focal AS build
+FROM swift:5.7.1-focal AS build
 
 WORKDIR /app
 RUN git clone --depth 1 --branch 1.20.2 https://github.com/apple/swift-protobuf --recursive
 WORKDIR /app/swift-protobuf
-RUN swift build -c release --static-swift-stdlib
+RUN swift build -c release --static-swift-stdlib -Xlinker -s
```

Comment:
Can you jot down why we had to add this `-Xlinker -s`, I think I understand .. but it'll be good to have this, even if it's because of an error switching to distroless.

Reply:
We didn't have to add this - it just strips the binaries to reduce the size (pretty significantly since we're statically linking in the swift stdlib).